charts/connect/values.yaml

```diff
@@ -295,10 +295,10 @@ operator:
     enabled: false
     # Additional annotations to be added to the PDB Operator
     annotations: {}
-    # Number of pods that are unavailble after eviction as number or percentage (eg.: 50%)
-    maxUnavailable: 1
     # Number of pods that are available after eviction as number or percentage (eg.: 50%)
-    minAvailable: 0
+    # minAvailable: 1
+    # Number of pods that are unavailble after eviction as number or percentage (eg.: 50%)
+    # maxUnavailable: 1
```

Comment:
`[question]`

Why have these defaults been removed? I'd consider this a breaking change if other Helm deployments relied on this.

Reply:
It is not possible to have both values at the same time in the `pdb` resource. This could be confusing for users, so I prefer that they use the feature they need. Keeping them as comments is helpful as an example.

>You can specify only one of maxUnavailable and minAvailable in a single PodDisruptionBudget. [[ref](https://kubernetes.io/docs/tasks/run-application/configure-pdb/#specifying-a-poddisruptionbudget)]

Reply:
I see your point. Based on the previous implementation it seems that even both were provided as defaults, only the `maxUnavailable` was set at any Helm deployment with the default value 1.

What I'm slightly concerned is that after this change, this will no longer be the case and could be breaking if users relied on this default configuration.

Reply:
You are right. So let me know what you prefer in this case. Would you like to keep it as it is in the main branch?

Reply:
I revert the change.